conanfile.py

```diff
@@ -5,7 +5,7 @@
 
 class HomestoreConan(ConanFile):
     name = "homestore"
-    version = "6.0.1"
+    version = "6.1.1"
```

Comment:
Technically a major (non-backwards compat) change. So if not going to bump Major, patch is good enough?